src/api/v1/statuses.ts

```diff
@@ -276,6 +277,26 @@ app.post(
         preferSharedInbox: true,
         excludeBaseUris: [new URL(c.req.url)],
       });
+      if (!owner.account.protected) {
```

Comment:
Not quite right: needs to take into account the privacy preferences of the account and the post.

So account should be discoverable and the post should be as:Public & not "quiet public" which I forget what that is really.

Reply:
I wasn't sure how to implement this correctly. What would be the correct way to check if we should forward the activity to the relay?

Reply:
Note that there's `account_owners.discoverable` column:

https://github.com/fedify-dev/hollo/blob/f985afe1cb70e499498a7cf90d3c2d7325e3de0b/src/schema.ts#L157

Also, you can check if `post.visibility === "public"`.